pom.xml

```diff
@@ -189,6 +193,33 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.pitest</groupId>
+          <artifactId>pitest-maven</artifactId>
+          <version>1.1.11</version>
+          <configuration>
+            <reportsDirectory>local/pit-reports</reportsDirectory>
+            <timestampedReports>false</timestampedReports>
+            <timeoutConstant>1000</timeoutConstant>
+            <targetClasses>
+              <param>${pitest.classes}</param>
```

Comment:
You are correct. `-DtargetClasses` works perfectly. I removed the property.

Now that we got rid of the optaplanner-specific property the Pitest plugin config can be moved to a higher level (kie-parent). Do you prefer that over having the config in optaplanner-parent only?